regress/cv32e40x_full.yaml

```diff
@@ -495,6 +495,12 @@ tests:
     cmd: make test TEST=zc_test
     num: 5
 
```

Comment:
b_ext_test runs only with b_ext_all config and not with the uvmt_cv32e40x_b_ext_abs. I dont know why, but will let the X version continue to include the b_ext_abs config. (This is not shown in the diff since cv32e40s_full.yaml seems to be to different from cv32e40x_full.yaml to be compared).
![image](https://github.com/openhwgroup/cv32e40x-dv/assets/110398284/eba8dd5f-9005-4ac6-a64b-bf66d843cc1e)

Reply:
I think it is fine to remove the b_ext_abs from the runs - I believe it was removed from the regressions as the b-ext test was updated with instructions that included the Zb*-extension(s) that was missing from the _abs configuration, and was no longer compatible.

Reply:
Okei, thank you. Then I will remove it.